bucket/dualmonitortaskbar.json

```
@@ -0,0 +1,28 @@
+{
+    "homepage": "http://sourceforge.net/projects/dualmonitortb/",
+    "version": "1.22.021813",
+    "license": {
+        "identifier": "GPL-3.0-or-later",
+        "url": "https://sourceforge.net/p/dualmonitortb/code/HEAD/tree/DualMonitorSolution/license.txt"
+    },
+    "description": "A taskbar for the second monitor.",
+    "url": "https://download.sourceforge.net/project/dualmonitortb/Releases/v01.22.021813/DualMonitor 1.22 Setup.exe",
```

Space needs to be encoded.

```suggestion
    "url": "https://download.sourceforge.net/project/dualmonitortb/Releases/v01.22.021813/DualMonitor%201.22%20Setup.exe",
```